docs/source/tutorials/quick-start.rst

```diff
@@ -12,12 +12,6 @@ GraphStorm is designed for easy-to-use GML models, particularly the graph neural
 
 This tutorial will use GraphStorm's built-in OGB-arxiv dataset for a node classification task to demonstrate these three steps.
 
-.. warning::
```

Comment:
Could you also help to remove the same warning section in the `own-data.rst` line 11-15?